lib/kitchen/driver/dokken.rb

```diff
@@ -59,8 +60,18 @@ class Dokken < Kitchen::Driver::Base
       default_config :volumes, nil
       default_config :write_timeout, 3600
 
+      def set_creds
+        if config[:creds_file]
+          @creds = JSON.parse(IO.read('config.json'))
```

Comment:
Shouldn't this be opening the `:creds_file` mentioned in the config? Not just always `config.json`

```suggestion
          @creds = JSON.parse(IO.read(config[:creds_file]))
```

Reply:
lol.  Good catch.